src/xmipp/libraries/tomo/tomo_detect_landmarks.h

```diff
@@ -0,0 +1,238 @@
+/***************************************************************************
+ *
+ * Authors:    Federico P. de Isidro Gomez			  fp.deisidro@cnb.csic.es
+ *
+ * Unidad de  Bioinformatica of Centro Nacional de Biotecnologia , CSIC
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+ * 02111-1307  USA
+ *
+ *  All comments concerning this program package may be sent to the
+ *  e-mail address 'xmipp@cnb.csic.es'
+ ***************************************************************************/
+
+#ifndef __TOMO_DETECT_LANDMARKS
+#define __TOMO_DETECT_LANDMARKS
+
+#include <iostream>
+#include <math.h>
+#include <limits>
+#include <complex>
+#include <string>
+#include <stdio.h>
+#include <core/xmipp_filename.h>
+#include <core/multidim_array.h>
+#include <core/xmipp_program.h>
+#include <core/metadata_label.h>
+#include <core/metadata_vec.h>
+#include <core/xmipp_image.h>
+#include <core/xmipp_fft.h>
+#include <core/xmipp_fftw.h>
+#include <core/xmipp_image_generic.h>
+#include <data/point3D.h>
+#include <data/point2D.h>
+#include <data/filters.h>
+#include <data/mask.h>
+#include <core/histogram.h>
+#include <fstream>
+   
+#define VERBOSE_OUTPUT
+
+// #define DEBUG_DIM
+// #define DEBUG_DOWNSAMPLE
+// #define DEBUG_INTERPOLATION_EDGES
+// #define DEBUG_SOBEL
+// #define DEBUG_CLOSING
```

Comment:
remove these comments

Reply:
This is commented on since it is only useful for debugging. In production, it is better to keep them commented to improve performance.